### PR TITLE
(GH-356) Get work items from build

### LINKED ISSUE #356: Option to get work items from a build
Please provide an option to get work items (ids) associated with a build.

### PR DESCRIPTION
**Pull request by twenzel**
Returns a list with work item ids associated with the build.
Fixes #356

### CODE REVIEW

**Review by pascalberger — COMMENTED** [review]
@twenzel Thank you very much for the pull request 👍 

With #361 I'm wondering if we should return a list of `AzureDevOpsWorkItem` instances instead of IDs?

**Comment by twenzel** [comment]
This feature (Returning WorkItems) will be added afterwards.

**Review by pascalberger — CHANGES_REQUESTED** [review]
The last commit enhances work item aliases. Can we please keep this PR focus on returning work items of a build and move enhancements to work item aliases to separate PRs

**Comment by twenzel** [comment]
@pascalberger I removed the last commit

**Comment by pascalberger** [comment]
> This feature (Returning WorkItems) will be added afterwards.

If there will be a method to return work items, what would be the advantage in also having a `GetWorkItemIds` method?

**Comment by twenzel** [comment]
> @twenzel If there will be a method to return work items, what would be the advantage in also having a GetWorkItemIds method?

It would be useful when you need to load any work items. The user might have an other source of work item ids than the build.
This is just the start of having work item tracking functionality in Cake.

**Comment by twenzel** [comment]
@pascalberger I can implement the Build.GetWorkItems method after the PR #364 was merged due to some changes on the WorkItem constructor.

**Comment by pascalberger** [comment]
> > @twenzel If there will be a method to return work items, what would be the advantage in also having a GetWorkItemIds method?
> 
> It would be useful when you need to load any work items. The user might have an other source of work item ids than the build. This is just the start of having work item tracking functionality in Cake.

Sorry, I still don't get it. When you read work items from somewhere else I don't see how it relates to reading work items from builds.

File: src/Cake.AzureDevOps/AzureDevOpsAliases.Pipelines.cs [hunk 1]
```diff
@@ -327,10 +328,106 @@ public static partial class AzureDevOpsAliases
...
+        ///
+        /// Information("Work item:");
+        /// foreach (var workItem in workItems)
+        /// {
+        ///     Information("  {0}: {1}", workItem.Id, workItem.Title);
+        /// }
+        /// ]]>
+        /// </code>
+        /// </example>
+        /// <returns>The work item ids associated with the build.
```
Comment (pascalberger) [inline src/Cake.AzureDevOps/AzureDevOpsAliases.Pipelines.cs:411]:
```suggestion
        /// <returns>The work items associated with the build.
```

**Review by pascalberger — APPROVED** [review]
LGTM beside some smaller typos

**Comment by pascalberger** [comment]
@twenzel your changes have been merged, thanks for your contribution 👍

**Pull request merged**